Wiredcraft/Section/Common/Model/WCBaseModel.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  WCBaseModel.swift
+//  Wiredcraft
+//
+//  Created by codeLocker on 2020/6/10.
+//  Copyright © 2020 codeLocker. All rights reserved.
+//
+
+import UIKit
+
+class WCBaseModel: NSObject {
```

Comment:
Why is there an empty class that is not being used anywhere?

Reply:
i was going to create some base classes,  to handle some same logic for his subclass.
in the test-mobile project, I didn't use it and forget delete it.